estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
Don't forget Unix conventions on text files' structure based on a line definition (cf. https://peps.python.org/pep-0008/ and https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)
```suggestion
from . import models

```

Please check all your files.